From 64669cf18b644d0d128ef6c5ef4098f56bcf9833 Mon Sep 17 00:00:00 2001
From: "J. Avila" <elavila@google.com>
Date: Sat, 7 Nov 2020 00:24:15 +0000
Subject: [PATCH 795/830] ANDROID: GKI: Enable CONFIG_RT_SOFTINT_OPTIMIZATION
X-NVConfidentiality: public

This config helps address audio buffer underrun issues on arm64 targets.

Bug: 168521633
Signed-off-by: J. Avila <elavila@google.com>
Change-Id: Idd0e71f2cb59c146e38c74c6f973bdc021d0f306
---
 arch/arm64/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 810b0481b52b..c83ace1982ac 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -26,6 +26,7 @@ CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_BPF=y
 CONFIG_NAMESPACES=y
 # CONFIG_PID_NS is not set
+CONFIG_RT_SOFTINT_OPTIMIZATION=y
 # CONFIG_RD_BZIP2 is not set
 # CONFIG_RD_LZMA is not set
 # CONFIG_RD_XZ is not set
-- 
2.17.1

